effector_mass: 0.2
f_offset:
    - -19.0
    - -4.0
    - 3.0
m_offset:
    - 0.0
    - 0.0
    - 0.0
r_offset:
    - 0.0
    - 0.0
    - 0.0
